packages/create-app/templates/default-app/packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -297,10 +297,13 @@ const groupPage = (
         <Grid item xs={12} md={6}>
           <EntityGroupProfileCard variant="gridItem" />
         </Grid>
+        <Grid item xs={12} md={6}>
+          <EntityLinksCard />
+        </Grid>
```

Comment:
Can you move this down? When we add things we try to keep the current state as much as possible. I also think Ownership being on the same row as the Profile Card makes more sense as it's more likely to have content in the early stages of adopting Backstage